openapi_server/apis/users_api.py

```diff
@@ -50,8 +50,10 @@ async def create_user(
     try:
         DbUser.get(create_user.username)
         return Response(status_code=409)
-    except Exception as e:
+    except DbUser.DoesNotExist:
         pass
+    except:
+        return Response(status_code=500)
```

Comment:
we would still need to respond with error message with 500 error

Reply:
Is including a `content=str(e)` in the `Response` parameters fine for that goal? This would assume that I change line 55 back to `except Exception as e:`. Other methods would be changing the response to be specifically JSONResponse instead of Response, or `raise HTTPException`.

Reply:
yep working for now. just chat with @valerieyang00 how we want to handle the errors in client side for HTTPException or JSONResponse.

Reply:
I'll add the `content` parameter and merge then? And I can bring up the options for client side use.

Reply:
Decided to go with raising `HTTPException` with the messages automatically put into json format as {"detail": "error message"}